warehouse/query-core/src/main/java/datawave/query/language/functions/jexl/Geowave.java

```diff
@@ -0,0 +1,264 @@
+package datawave.query.language.functions.jexl;
+
+import datawave.query.language.functions.QueryFunction;
+import datawave.webservice.query.exception.BadRequestQueryException;
+import datawave.webservice.query.exception.DatawaveErrorCode;
+
+import java.text.MessageFormat;
+import java.util.ArrayList;
+import java.util.List;
+
+public interface Geowave {
+    
+    class Contains extends JexlQueryFunction {
```

Comment:
Lets consolidate some of this code: e.g. make an abstract class in the middle that gives us the validate and toString method implementations that are duplicated in each one of these.

Reply:
Can you add interface default methods for validate and toString so we don't need another class?

Reply:
Fixed!